shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -121,13 +124,16 @@ public void writeFields(DataOutput out) throws IOException {
             out.writeLong(requestId);
             out.writeUTF(segment);
             out.writeUTF(correctHost);
+            out.writeUTF(serverStackTrace);
         }
 
         public static WireCommand readFrom(DataInput in, int length) throws IOException {
             long requestId = in.readLong();
             String segment = in.readUTF();
             String correctHost = in.readUTF();
-            return new WrongHost(requestId, segment, correctHost);
+            String serverStackTrace = (length > Long.BYTES + 2 * UTF_HEADER_BYTES + segment.getBytes().length + correctHost.getBytes().length) ?
```

Comment:
` ((ByteBufInputStream) in).available()` this would return the remaining bytes too right?
(ref: `io.pravega.shared.protocol.netty.WireCommandType#readFrom` this takes ByteBufInputStream as an input)